### PR TITLE
Add e2e into CI

### PR DESCRIPTION
**Pull request by Yisaer**
### What problem does this PR solve? 
Add e2e into ci

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

File: ci/pingcap_chaos_mesh_build_kind.groovy [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+import groovy.transform.Field
+
+@Field
+def podYAML = '''
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    # we pretend as tidb-operator in order not to meet tidb-operator-e2e job in the same node
+    app: tidb-operator-e2e
```
Comment (Yisaer) [inline ci/pingcap_chaos_mesh_build_kind.groovy:14]: we would share same cluster with tidb-operator-e2e, so we pretend ourselves like tidb-operator-e2e in order not to run both kind of e2e job in one node.

File: test/e2e/config/config.go [hunk 2]
```diff
@@ -0,0 +1,31 @@
+// Copyright 2019 PingCAP, Inc.
```
Comment (fewdan) [inline test/e2e/config/config.go:1]: s/2019/2020/g

Comment (Yisaer) [reply test/e2e/config/config.go]: updated.

**Pull request merged**